library/ssl_tls13_client.c

```diff
@@ -58,35 +60,37 @@ static int ssl_tls13_write_supported_versions_ext( mbedtls_ssl_context *ssl,
      * - extension_type         (2 bytes)
      * - extension_data_length  (2 bytes)
      * - versions_length        (1 byte )
-     * - versions               (2 bytes)
+     * - versions               (2 to 4 bytes)
```

Comment:
Let's keep it simple. I know it works only in TLS 1.2 + TLS 1.3 case but I think it will last for quite a long time. When there will be a new version of TLS we will think about it.

Reply:
I agree. More generally, I think comments only need to be correct for the code as it is now - we can update them when we change the code. Also predicting the future is hard: the way versions are offered is completely different in 1.3 from what it was throughout history (it remained the same from SSL 3.0 to TLS 1.2), so who knows what TLS 1.4 or 2.0 will look like? (Well, in this particular case, since the mechanism was just changed, hopefully it will not change again soon, but this is just to illustrate my general point that we should focus on the present.)